src/main/java/model/data/DataModel.java

```diff
@@ -2,6 +2,8 @@
 
 import java.util.*;
 
+import static java.util.Collections.unmodifiableList;
```

Comment:
Ik vind dit niet heel mooi, je gebruikt hem maar 1 keer, dus dan is de winst die je boekt op het sneller typen en sneller lezen van unmodifiableList niet groter dan de energie die het mij kost om te bedenken dat unmodifiableList in Collections zit en dat je dus een static import doet.